core/src/main/scala/cats/AutoVariance.scala

```diff
@@ -0,0 +1,13 @@
+package cats
+
+trait AutoVariance extends AutoVarianceLowPriority {
```

Comment:
Does this need to be in the public API? It kind of makes me think we should have a `conversions` package along with `syntax` and `instances` (especially since the root `cats` package is already pretty cluttered).